config/environment.js

```diff
@@ -0,0 +1,14 @@
+const dotenv = require('dotenv');
+const requireEnv = require('require-environment-variables');
+
+function validate() {
```

Comment:
Strictly speaking this is configuring (the 2nd part is configuring validation).

Bit pedantic but then so is lint :)

Also, giving the new GitHub review process a spin.